libs/components/layout/src/lib/modules/toolbar/toolbar-item.component.scss

```diff
@@ -1,27 +1,73 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides('.sky-toolbar-item') {
+  --sky-override-toolbar-item-spacing-bottom: #{$sky-padding-half};
+  --sky-override-toolbar-item-spacing-right: #{$sky-margin-half};
+}
+
+@include compatMixins.sky-modern-overrides('.sky-toolbar-item') {
+  --sky-override-toolbar-item-text-color: var(--sky-color-text-default);
+  --sky-override-toolbar-search-icon-color: var(--sky-color-icon-default);
+}
 
 .sky-toolbar-item {
-  margin-right: $sky-margin-half;
-  margin-bottom: $sky-padding-half;
+  margin-right: var(
+    --sky-override-toolbar-item-spacing-right,
+    var(--sky-space-gap-action_group-m)
+  );
+  margin-bottom: var(--sky-override-toolbar-item-spacing-bottom);
+
+  ::ng-deep
+    sky-search
```

Comment:
Should we try to add this to the search styles somehow so that we aren't using `ng-deep`? I know Paul has been pretty against that recently and this is adding a new non-existing `ng-deep`.